test/integration/targets/vmware/tasks/setup.yaml

```diff
@@ -0,0 +1,74 @@
+---
+- name: start vcsim
+  uri:
+    url: http://{{ vcsim }}:5000/spawn?cluster=2
```

Comment:
```suggestion
    url: http://{{ vcsim }}:5000/spawn?datacenter=2&cluster=2&folder=2&ds=2&pool=2
```

Reply:
I modified them a bit to avoid the constraint and they are all passing.

Reply:
to elaborate a bit, I would prefer to reduce the "default" environment because we need to re-prepare it before each test. This can quickly be really time consuming on really HW.
For the few cases where it matters, it's better to add the extra resources in the test itself.